main/src/ca/uwaterloo/flix/api/lsp/provider/SemanticTokensProvider.scala

```diff
@@ -521,10 +521,10 @@ object SemanticTokensProvider {
     case Expr.Throw(exp, _, _, _) =>
       visitExp(exp)
 
-    case Expr.TryWith(exp, eff, rules, _, _, _) =>
+    case Expr.Handler(eff, rules, _, _, _, _, _) =>
```

Comment:
eff -> symUse

Reply:
The other things use `sym` for their `SymUse` things so I went with that